src/components/Graph/canvas.js

```diff
@@ -73,6 +73,7 @@ export default class Canvas {
     this.scene = new Scene();
     this.scene.background = new Color(0x5f6b7a);
     this.camera = new PerspectiveCamera(75, width / height, 0.1, 1000);
+    this.camera.fov = 60;
```

Comment:
Why this change? (And if we're changing the field of view here , we might as well just do it in the constructor above.)

Reply:
"I noticed a line was still disappearing right now the bottom on that major zoom, so I slightly tweaked the FOV to hide it" :)

And yep will do it in the constructor

Reply:
OK done!